schema/deploy/shipping/views.sql

```diff
@@ -3129,16 +3129,23 @@ create materialized view shipping.__uw_encounters as (
-    , (select boolean_response from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'screen_positive') as screen_positive
-    , (select boolean_response from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'daily_symptoms') as daily_symptoms
-    , (select case string_response[1] when 'yes' then true else false end from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'daily_exposure') as daily_exposure
-    , (select string_response[1] from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'daily_exposure_known_pos') as daily_exposure_known_pos
-    , (select boolean_response from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'testing_trigger') as testing_trigger
-    , (select boolean_response from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'surge_selected_flag') as surge_selected_flag
+    , q1.boolean_response as screen_positive
+    , q2.boolean_response as daily_symptoms
+    , case when q3.encounter_id is null then null when q3.string_response[1] = 'yes' then true else false end as daily_exposure
+    , q4.string_response[1] as daily_exposure_known_pos
+    , q5.boolean_response as testing_trigger
+    , q6.boolean_response as surge_selected_flag
     , (select max(collected)::date from warehouse.sample where encounter_id = encounter.encounter_id group by encounter_id) as sample_collection_date
-    , (select date_response[1]::date from shipping.fhir_questionnaire_responses_v1 where encounter_id = encounter.encounter_id and link_id = 'prior_test_positive_date') as prior_test_positive_date
+    , q7.date_response[1]::date as prior_test_positive_date
 	from warehouse.encounter
 	join warehouse.individual using (individual_id)
+	left join shipping.fhir_questionnaire_responses_v1 q1 on q1.encounter_id = encounter.encounter_id and q1.link_id = 'screen_positive'
+	left join shipping.fhir_questionnaire_responses_v1 q2 on q2.encounter_id = encounter.encounter_id and q2.link_id = 'daily_symptoms'
+	left join shipping.fhir_questionnaire_responses_v1 q3 on q3.encounter_id = encounter.encounter_id and q3.link_id = 'daily_exposure'
+	left join shipping.fhir_questionnaire_responses_v1 q4 on q4.encounter_id = encounter.encounter_id and q4.link_id = 'daily_exposure_known_pos'
+	left join shipping.fhir_questionnaire_responses_v1 q5 on q5.encounter_id = encounter.encounter_id and q5.link_id = 'testing_trigger'
+	left join shipping.fhir_questionnaire_responses_v1 q6 on q6.encounter_id = encounter.encounter_id and q6.link_id = 'surge_selected_flag'
+	left join shipping.fhir_questionnaire_responses_v1 q7 on q7.encounter_id = encounter.encounter_id and q7.link_id = 'prior_test_positive_date'
```

Comment:
(non blocker) it will result in a little more text, but naming the tables by the link id may be more informative here - e.g., instead of 

`left join shipping.fhir_questionnaire_responses_v1 q1 on q1.encounter_id = encounter.encounter_id and q1.link_id = 'screen_positive'`

`left join shipping.fhir_questionnaire_responses_v1 q_screen_positive on q_screen_positive.encounter_id = encounter.encounter_id and q_screen_positive.link_id = 'screen_positive'`

Reply:
Good suggestion, @kschwabefry! I've made that change.